docs/design/Percona-DBaaS-Platform-design.md

```diff
@@ -0,0 +1,154 @@
+-->
+
+Percona DBaaS Platform OLM Catalog is a selection of operators that are tested and validated for Percona DBaaS products.
+
+## Motivation
+
+<!--
+This section is for explicitly listing the motivation, goals, and non-goals of
+this KEP.  Describe why the change is important and the benefits to users. The
+motivation section can optionally provide links to [experience reports] to
+demonstrate the interest in a KEP within the wider Kubernetes community.
+
+[experience reports]: https://github.com/golang/go/wiki/ExperienceReports
+-->
+
+To create well tested and well maintained catalog to enable Percona platform in k8s.
+
+## Proposal
+
+<!--
+This is where we get down to the specifics of what the proposal actually is.
+This should have enough detail that reviewers can understand exactly what
+you're proposing, but should not include things like API designs or
+implementation. What is the desired outcome and how do we measure success?.
+The "Design Details" section below is for the real
+nitty-gritty.
+-->
+
+Should be an implementation of a platform as described in a [design document](dbaas-catalog-design.md#create-new-platform).
+
+## Design Details
+
+<!--
+This section should contain enough information that the specifics of your
+change are understandable. This may include API specs (though not always
+required) or even code snippets. If there's any ambiguity about HOW your
+proposal will be implemented, this is the place to discuss them.
+-->
+
+Create the branch `percona-platform` that would be as close as possible to a `main` but would have additional set of tests and might include/exclude some of the operators.
+
+Operator's team creates PR (manually or automatically with CI) and changes first land to `main` and then merged to the `percona-platform`. Merge to `percona-platform` happens either automatically or manually by owners (after some quality gate).
+
+Create `docker.io/percona-platform` registry to store platform images. Bundles use same operator's name (for example `quay.io/operatorhubio/percona-server-mongodb-operator:v1.13.0`) so pushing it to the `percona` or `perconalab` might introduce more misunderstanding and confusions what is operator image and what is a bundle.
+
+Images would be created automatically by CI in `dbaas-catalog`:
+  - with any commit to `percona-platform`: docker.io/percona-platform/dbaas-catalog:main
+  - when new tag pushed to `percona-platform` branch:
+    - `docker.io/percona-platform/dbaas-catalog:latest`
+    - `docker.io/percona-platform/dbaas-catalog:vX.Y.Z`
+
+Images required from operators are bundle image:
+  - **Candidate** Channel: `docker.io/percona-platform/operatorA:vX.Y.Z-rcN`
+  - **Stable** Channel: `docker.io/percona-platform/operatorA:vX.Y.Z`
+
+### Owners
+
+Platform owners are responsible and accountable for the platform CI/CD, quality (green actions) and state of the operators (new versions) in the catalog.
+
+Platform owners:
+  - @denisok
+  - @nmarukovich
```

Comment:
@denisok could you add one more person to owner list? I could be @cap1984. Do you plan to add pg operator?